web/components/App.jsx

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import _ from 'lodash';
+import FilterHeader from './FilterHeader.jsx';
+import EventMessage from './EventMessage.jsx';
+import UserMessage from './UserMessage.jsx';
+import ChatMessage from './ChatMessage.jsx';
+import AttachmentMessage from './AttachmentMessage.jsx';
+import ToastMessage from './ToastMessage.jsx';
+import ChatBox from './ChatBox.jsx';
```

Comment:
@kriscfoster do you need the '.jsx' extension here?

Reply:
I was getting a webpack exception when `.jsx` was omitted, maybe it's something to do with our webpack config file. Will take a look

Reply:
It's not a problem really, I just thought maybe it was possible to omit it